pandas/core/generic.py

```diff
@@ -3672,6 +3672,17 @@ def to_csv(
         r"""
         Write object to a comma-separated values (csv) file.
```

Comment:
I think CSV should be capitalized.
```suggestion
        Write object to a comma-separated values (CSV) file.
```